### PR TITLE
Adds collection detail component

### LINKED ISSUE #5: Create Accordion component
## Describe the solution you'd like
A group of expandable/collapsable panels. Allow multiple panels to be opened at the same time.

## Additional context
Note that the `<summary>` element shouldn’t be used for disclosure if it has a heading inside of it, because it will prevent the heading from showing up for SRs when navigating by heading (this is also true for headings inside of buttons).

This component should be created with consideration/in conversation with a Tree component (issue #21 ) in thinking about the best strategy for handling nested collection/object navigation.

### LINKED ISSUE #109: Build out collection detail component
## Is your feature request related to a problem? Please describe.
Build out collection detail component as specified in design treatments.

### PR DESCRIPTION
**Pull request by helrond**
Adds a collection detail component. I'm thinking this can definitely be combined with the object detail component, but I'll do that in a separate PR. fixes #109 

Also adds accordion component, fixes #5

### CODE REVIEW

**Comment by HaSistrunk** [comment]
This looks good! I adjusted the background to be off-white to align with changes in PR #115 , if you're good with that.

**Review by HaSistrunk — APPROVED** [review]

**Comment by helrond** [comment]
Great, thanks! I'll wait until #115 is merged and double-check everything is cool.

**Pull request merged**
